README.md

```diff
@@ -123,10 +123,14 @@ As we use Terraform to deploy our infrastructure. Our BASH deployment scripts ar
 
 ### Get BASH script parameters
 
-We always use the terraform step to launch as an argument of our BASH infrastructure script. So, the script `terraform/get-script-parameters.sh` checks if the argument exists and get arguments in the right variable.
+You can use the script `components/check-bash-parameters.sh` to parse scripts arguments. By default it will wait 1 argument and export it as `STEP` variable.
```

Comment:
A part for this script will be better I think, It can be used alone and it is very useful not only for Terraform